NewForce-Cohort-4/nutshell-frugal-badgers
# Jn chat rt

# Description
Enabled real-time chat between multiple users when logged in. Commented additional functions.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
Open two browser windows, log into two separate user accounts. Set chat to online. Send message. The alternate user will see their messages update in real-time.
# Checklist:
